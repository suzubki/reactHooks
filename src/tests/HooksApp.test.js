import HooksApp from "../HooksApp";
import { shallow } from "enzyme";

test("should render <HooksApp></HooksApp>", () => {
    const wrapper = shallow(<HooksApp />);

    expect(wrapper).toMatchSnapshot();
});
