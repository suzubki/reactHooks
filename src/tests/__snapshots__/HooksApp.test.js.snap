// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <HooksApp></HooksApp> 1`] = `
<div>
  Hola mundo
</div>
`;
